src/main/java/com/example/onlinebookstore/model/Category.java

```diff
@@ -0,0 +1,27 @@
+package com.example.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Table(name = "categories")
+@Data
```

Comment:
as I know it`s better to use @Getter and @Setter instead @Data, but it`s not mistake

Reply:
in this case, it is ok. A problem can occur if the entity has a relation